src/main/java/JavaLessons/Lesson1/Lesson3.java

```diff
@@ -0,0 +1,222 @@
+            if (arr[i] < 6) {
+                arr[i] = arr[i] * 2;
+            }
+            System.out.print(arr[i]);
+            if (i < arr.length -1) {
+                System.out.print(", ");
+            }
+        }
+        System.out.print("}\n");
+    }
+// Создать квадратный двумерный целочисленный массив (количество строк и столбцов одинаковое), и с помощью цикла(-ов) заполнить его диагональные элементы единицами (можно только одну из диагоналей, если обе сложно). Определить элементы одной из диагоналей можно по следующему принципу: индексы таких элементов равны, то есть [0][0], [1][1], [2][2], …, [n][n];
+public static void crossArray() {
+    int[][] arr = new int[5][5];
+    for (int i = 0; i < arr.length; i++) {
+      for (int j = 0; j < arr.length; j++) {
+          if ((i == j) || (i == arr.length - 1 - j)) {
+              arr[i][j] = 1;
+          }
+          System.out.print(arr[i][j] + " ");
+      }
+        System.out.println();
+    }
+}
+// Написать метод, принимающий на вход два аргумента: len и initialValue, и возвращающий одномерный массив типа int длиной len, каждая ячейка которого равна initialValue;
+public static void lenArray(int len, int initialValue) {
+        int[] arr = new int[len];
+        for (int i = 0; i < arr.length; i++) {
+          arr[i] = initialValue;
+            System.out.print(arr[i] + " ");
+        }
+    System.out.println();
+}
+// Задать одномерный массив и найти в нем минимальный и максимальный элементы ;
+    public static int[] minMaxArray(int length) {
+        int[] arr = new int[length];
+        Random random = new Random();
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = random.nextInt();
+        }
+        Arrays.sort(arr);
+        System.out.println(Arrays.toString(arr));
+        return arr;
+    }
+    public static int[] minMaxArrayV2(int length) {
+        int[] arr = new int[length];
+        Random random = new Random();
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = random.nextInt();
+        }
+        System.out.println(Arrays.toString(arr));
+        int max = arr[0];
+        int min = arr[0];
+        for (int i = 0; i < arr.length; i++) {
+            if (arr[i] > max) {
+                max = arr[i];
+                }
+            else if (arr[i] < min) {
+                min = arr[i];
+            }
+        }
+        System.out.println("Max is " + max + ". Min is " + min);
+        return arr;
+    }
+// Написать метод, в который передается не пустой одномерный целочисленный массив, метод должен вернуть true, если в массиве есть место, в котором сумма левой и правой части массива равны.
+    public static boolean sumTwoPartArray(int ... arr) {
+        int totalSum = 0;
+        for (int num : arr) {
+            totalSum += num;
+        }
+        int leftSum = 0;
+        for (int i = 0; i < arr.length; i++) {
+            totalSum -= arr[i];
+            leftSum += arr[i];
```

Comment:
Correct.
However too many actions. YOu could just calculate that half and compare with it